samples/snippets/src/main/java/com/example/spanner/PgSpannerSample.java

```diff
@@ -577,6 +577,86 @@ static void writeUsingDml(DatabaseClient dbClient) {
   }
   // [END spanner_postgresql_dml_getting_started_insert]
 
+  // [START spanner_postgresql_insert_dml_returning]
+  static void insertUsingDmlReturning(DatabaseClient dbClient) {
+    // Insert records into SINGERS table and returns the
+    // generated column FullName of the inserted records
+    // using ‘RETURNING FullName’.
+    // It is also possible to return all columns of all the
+    // inserted records by using ‘RETURNING *’.
+    dbClient
+        .readWriteTransaction()
+        .run(transaction -> {
+          String sql =
+              "INSERT INTO Singers (SingerId, FirstName, LastName) VALUES "
+                  + "(12, 'Melissa', 'Garcia'), "
+                  + "(13, 'Russell', 'Morales'), "
+                  + "(14, 'Jacqueline', 'Long'), "
+                  + "(15, 'Dylan', 'Shaw') RETURNING FullName";
+          try (ResultSet resultSet = transaction.executeQuery(Statement.of(sql))) {
```

Comment:
nit: I think it's worth adding a small comment in the code both here and in the other samples that users must use `transaction.executeQuery(..)` for DML with `RETURNING` clauses.

Reply:
Done.